### ISSUE TITLE
DSL Query meta field `__typename`

### ISSUE THREAD

**Issue by ThiloAschebrock** [graphql-python/gql#250]
It should possible to query a meta field like `__typename` with the DSL module. 

After trying to do that, looking at the documentation and the code, I come to the conclusion that it is not possible to query `__typename` with DSL.

Could you please either tell me how to query `__typename`  or provide the possibility to do so?

**Comment by leszekhanusz** [comment]
This should be made easier but you should be able to do it now with something like this:

```python                                                                        
def test_hero_name_query_with_typename(ds):                                         
    from gql.dsl import DSLField                                                                                        
    from graphql import GraphQLScalarType                                                                               
                                                                                                                        
    query = """                                                                                                         
hero {                                                                                                                  
  name                                                                                                                  
  __typename                                                                                                            
}                                                                                                                       
    """.strip()                                                                                                         
                                                                                                                        
    typename_field = DSLField("__typename", GraphQLScalarType("meta"), None)                                            
    query_dsl = ds.Query.hero.select(ds.Character.name, typename_field)                                                 
                                                                                                                        
    assert query == str(query_dsl)
```

**Comment by leszekhanusz** [comment]
This is now implemented properly in the PR #259
